### ISSUE TITLE
Homepage header text

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/COLOGNE#232]
We have:
```
This web page provides access to many of the Sanskrit lexicons prepared by the Institute of Indology and Tamil Studies, Cologne University.
The dictionaries are organized primarily by the secondary language (English, German, etc.), and then by date of publication.
Each dictionary has several types of display, as well as a comprehensive selection of materials for download. 
```

Now the text looks like a pyramid, great. But it was for years I was thinking how to rewrite it.

> This web page provides

Can be left out.

> access to many of the Sanskrit lexicons

Not many, most, or almost all

> prepared by the Institute of Indology and Tamil Studies

Not only from the ones prepared, but from all that ever existed.

Before:
`This web page provides access to many of the Sanskrit lexicons prepared by the Institute of Indology and Tamil Studies, Cologne University.`

After:
`Since 1994 Sanskrit lexicons prepared by the Institute of Indology and Tamil Studies, Cologne University.`

Before:
`The dictionaries are organized primarily by the secondary language (English, German, etc.), and then by date of publication. `

After:
`The 36 dictionaries are organized primarily by the secondary language (English, German, etc.), and then by date of publication (1832 till 1976).`

Before:
`Each dictionary has several types of display, as well as a comprehensive selection of materials for download. `

After:
`Each dictionary has several types of display (B L A M), as well as PDF scan and XML (in SLP1) files for download (D). `

Comment:
1) (B L A M) links to a video instead of documentation? I can make a video if Jim agrees
2) SLP1 links to Wikipedia

**Comment by funderburkjim** [comment]
Several of the changes are now installed in a test version: [index_test.html](http://www.sanskrit-lexicon.uni-koeln.de/index_test.html).

The first sentence was altered slightly from the suggestion, so that it would be idiomatic English.

There is also a link to the Wikipedia page for SLP1 --- I didn't know that this Wikipedia page existed.  Good addition.

I would be glad to put a link to a video tutorial regarding usage of 'B L A M'.  On

**Comment by funderburkjim** [comment]
###  repository for sanskrit-lexicon home page construction

A [sanlexhome](https://github.com/funderburkjim/sanlexhome) repository contains the code
which generates the home page for sanskrit-lexicon.uni-koeln.de web site.

**Comment by gasyoun** [comment]
> Several of the changes are now installed in a test version: index_test.html.

Great, but what about you? Please add a link to a photo and at least a one-paragraph bio about yourself.

**Comment by funderburkjim** [comment]
These changes now installed as index.html, the live home page.

I think there should be some link to the android versions @drdhaval2785 made in conjunction with @vvasuki .    Maybe someone can prepare a link target, and we can decide how the link to this target
should be integrated on the sanskritlexicon home page.

**Comment by drdhaval2785** [comment]
https://github.com/indic-dict/stardict-sanskrit/tree/master/sa-head for Sanskrit headwords
https://github.com/indic-dict/stardict-sanskrit/tree/master/en-head for English headwords

**Comment by gasyoun** [comment]
Thanks, @drdhaval2785 , good to see you back.

**Comment by funderburkjim** [comment]
These links are not quite what we need for a link on Cologne homepage.
I was thinking of a link for *Android versions of Cologne dictionaries*.   The target would be a user-friendly guide to getting the Android versions of the Cologne dictionaries.
Perhaps such a guide is already prepared?

**Comment by gasyoun** [comment]
> Perhaps such a guide is already prepared?

No. First we need a total UI remake. After no guide will be needed,

**Comment by drdhaval2785** [comment]
@gasyoun would not be happy with any text. Closing.

**Comment by gasyoun** [comment]
> would not be happy with any text

Not exactly, we'll work on it again soon.

@funderburkjim `The 36 dictionaries are organized` -> `The 37 dictionaries are organized`

**Comment by funderburkjim** [comment]
Change 36 to 37 ... Done.

> (B L A M) links to a video instead of documentation? I can make a video if Jim agrees

OK with me.  We can change text after we see what the video does.

**Comment by funderburkjim** [comment]
@drdhaval2785 

The Cologne homepage mention of Android versions is currently:

>Most of the dictionaries are also available for offline usage in android phones via 
[this application](https://play.google.com/store/apps/details?id=sanskritcode.sanskritdictionaryupdater&hl=en).


This is too spartan, I think.
Better would be link to a separate new web page (such as
 https://sanskrit-lexicon.github.io/android.html or
 https://github.com/sanskrit-lexicon/cologne/android.md).
This web page would:
* mention the Google Play Store app
* Provide a tutorial for novice Android users describing steps required
   * installation of Colordict
   * installation of Sanskrit dictionaries
   * making Sanskrit dictionaries available in Colordict
   * Using Colordict to look up a word in the dictionary, including spelling conventions
      and how to bypass the built-in autocomplete features of Android text input.
* mention the development urls cited in the [above comment](https://github.com/sanskrit-lexicon/COLOGNE/issues/232#issuecomment-453808067)

When I originally tried to use the Sanskrit dictionaries on Android,  I very greatly felt
the need for a tutorial -- probably some other occasional Android users would also
appreciate such a tutorial.

**Comment by drdhaval2785** [comment]
Let's become a generous athenian and provide the tutorial. I am completely in agreement. Will try to write a documentation for the same.

**Comment by gasyoun** [comment]
> Will try to write a documentation for the same.

Thanks, and we will make it better after together ))

**Comment by vvasuki** [comment]
> Let's become a generous athenian and provide the tutorial. I am completely in agreement. Will try to write a documentation for the same.

@drdhaval2785 There already exists some tutorial as to how stardict dictionaries are to be used on Android devices - https://sanskrit-coders.github.io/dictionaries/offline/#how-to-install-and-use-dictionaries-on-your-device . You can just link to it, and edit it as appropriate. That way you won't have to edit in multiple place if some installation method becomes obsolete and a new one arises.

**Comment by drdhaval2785** [comment]
Great. Will do so.

**Comment by gasyoun** [comment]
> That way you won't have to edit in multiple place if some installation method becomes obsolete and a new one arises.

Generally very well written, thanks. Let us get to the details.

Let us move to some general wiki place? What is the date, @vvasuki it was last updated? Why not screenshots or videos?

I dislike the transliteration scheme a lot: rAmAyaNa-vyAkhyA-koshaH. Hope Dhaval as well ))

**Comment by vvasuki** [comment]
> Let us move to some general wiki place? 

It's already a "general wiki place" :-) You edit by clicking on the pencil icon shown below and sending a pull request.
![image](https://user-images.githubusercontent.com/2664797/103006365-32ffde00-4558-11eb-89a3-751b39f1154e.png)


> What is the date, @vvasuki it was last updated? 

Github has commit history you can refer to. Like so -
![image](https://user-images.githubusercontent.com/2664797/103006802-5460ca00-4558-11eb-8d71-2c55d6c4c529.png)

> Why not screenshots or videos?

Good to have it as well (there's a video in play store listing IIRC, and some handouts linked ). @drdhaval2785 might as well move and further develop the material in a fresh page (for whatever good it does) in the same site.
